src/components/signup/reducers.js

```diff
@@ -0,0 +1,31 @@
+import {
+    SET_USER, SET_ERRORS, CLEAR_ERRORS, LOADING_UI, SET_AUTHENTICATED, SET_UNAUTHENTICATED
+} from './actionType';
+
+const initialState = {
+    authenticated: false,
+    errors: {},
+    loading: false,
+    token: '',
+    user: {},
+};
+export default (state = initialState, { type, payload }) => {
+    switch (type) {
+        case SET_USER:
+            return { ...state, loading: false, user: payload };
+        case SET_AUTHENTICATED:
+            localStorage.setItem('FBToken', payload);
```

Comment:
We can use cookies instead of localstorage.

Reply:
Is there any superior advantage of cookies over local storage ?